### PR TITLE
[frontend] Add defaultStartTime and stopTime for StixCoreRelationship creation (#8575)

### PR DESCRIPTION
**Pull request by marieflorescontact**
### Proposed changes

* add default value for relation ship creation from knowledge graph and investigation graph
* remove 1 minute to make sure startime and stoptime are different

### Related issues

* #https://github.com/OpenCTI-Platform/opencti/issues/8575

### Checklist

- [x] I consider the submitted work as finished
- [x] I tested the code for its functionality
- [ ] I wrote test cases for the relevant uses case (coverage and e2e)
- [ ] I added/update the relevant documentation (either on github or on notion)
- [ ] Where necessary I refactored code to improve the overall quality

### CODE REVIEW

File: opencti-platform/opencti-front/src/utils/Time.js
Comment (lndrtrbn) [inline opencti-platform/opencti-front/src/utils/Time.js]: In the issue Nino ask for one second less instead of one minute, was there any changes?

File: opencti-platform/opencti-front/src/utils/Time.js
Comment (lndrtrbn) [inline opencti-platform/opencti-front/src/utils/Time.js]: It would be better to have a generic function allowing to subtract `x` seconds from a date instead of a very specific function. You have an example in this same file with function `minutesBefore`

File: opencti-platform/opencti-front/src/private/components/analyses/groupings/GroupingKnowledgeGraphBar.jsx
Comment (lndrtrbn) [inline opencti-platform/opencti-front/src/private/components/analyses/groupings/GroupingKnowledgeGraphBar.jsx]: Calling 2 times function `now` results having two different dates. Yes the diff would be only some milliseconds. But I think it's better to have only one call to `now` to be sure we have the same date. (Same remark for other files)

File: opencti-platform/opencti-front/src/utils/Time.js
Comment (marieflorescontact) [reply opencti-platform/opencti-front/src/utils/Time.js]: because second was not taken into account

Comment (marieflorescontact) [reply opencti-platform/opencti-front/src/utils/Time.js]: I checked with @nino-filigran . 1 minute is ok.
we don't display second tht is why i used minute.

File: opencti-platform/opencti-front/src/utils/Time.js
Comment (marieflorescontact) [reply opencti-platform/opencti-front/src/utils/Time.js]: As we could remove 1 minute (instead of 1 seconde as I mentioned above), I used `minutesBefore` that was already existing and I deleted `validStartTimeForRelationCreation `

**Comment by lndrtrbn** [comment]
Missing default dates when creating relationships from:
- Malware analyses / Knowledge
- Incidents / Knowledge
- Observables / Knowledge / Relations
- Artifacts / Knowledge / Relations
- Indicators / Knowledge
- Infrastructures / Knowledge
- Well all entities that use the same component for create relationships than the ones above

Default start and stop are identical in:
- Observables / Knowledge / Nested objects
- Artifacts / Knowledge / Nested objects

**Comment by marieflorescontact** [comment]
> Missing default dates when creating relationships from:
> 
> * Malware analyses / Knowledge
> * Incidents / Knowledge
> * Observables / Knowledge / Relations
> * Artifacts / Knowledge / Relations
> * Indicators / Knowledge
> * Infrastructures / Knowledge
> * Well all entities that use the same component for create relationships than the ones above
> 
> Default start and stop are identical in:
> 
> * Observables / Knowledge / Nested objects
> * Artifacts / Knowledge / Nested objects

ok you are right, I checked with product team, even if it is a bigger scope than the initial issue, we want to add default start time and stop time everywhere relation creation is allowed.

**Comment by marieflorescontact** [comment]
PR status: all comments have been taken into account.
waiting for 2nd round of review or approval .

**Review by lndrtrbn — APPROVED** [review]

**Pull request merged**
